src/lib/annotations/drawing/DrawingModeController.js

```diff
@@ -96,6 +97,34 @@ class DrawingModeController extends AnnotationModeController {
     }
 
     /**
+     * Unbind drawing mode listeners. Resets the undo and redo buttons to be disabled if they exist
+     *
+     * @inheritdoc
+     * @protected
+     * @return {void}
+     */
+    unbindModeListeners() {
+        super.unbindModeListeners();
+
+        annotatorUtil.disableElement(this.undoButtonEl);
+        annotatorUtil.disableElement(this.redoButtonEl);
+    }
+
+    /**
+     * Deselect a saved and selected thread
+     *
+     * @return {void}
+     */
+    cleanSelector() {
```

Comment:
"clean" doesn't really mean much to me, maybe destroy or remove?

Reply:
changed to removeSelection